generators/client-2/templates/src/main/webapp/app/blocks/config/_uib-pager.config.ts

```diff
@@ -1,9 +1,9 @@
 import { ITEMS_PER_PAGE } from '../../shared';
 
-PagerConfig.$inject = ['uibPagerConfig'];
+//PagerConfig.$inject = ['uibPagerConfig'];
```

Comment:
I don't exactly understand what you mean @sendilkumarn, atm i'm working on migrating the interceptors to Angular2 but I can't see whats the link between this and our interceptors.